concent_api/core/views.py

```diff
@@ -231,22 +306,27 @@ def handle_send_reject_report_computed_task(client_message):
             client_message.TYPE,
             force_report_computed_task.task_to_compute.compute_task_def['task_id'],
             client_message.serialize(),
-            status = ReceiveStatus
+            provider_public_key  = force_report_computed_task_from_database.last().auth.provider_public_key_bytes,
+            requestor_public_key = client_public_key,
+            status               = ReceiveStatus
         )
         return HttpResponse("", status = 202)
     else:
         raise Http400("Time to acknowledge this task is already over.")
 
 
-def handle_send_force_get_task_result(client_message: message.concents.ForceGetTaskResult) -> message.concents:
+def handle_send_force_get_task_result(request, client_message: message.concents.ForceGetTaskResult) -> message.concents:
```

Comment:
Please fix the type annotation while you're at it. I think I reported it at some point but it's still broken.

Reply:
I would like to keep it as a reminder that it require further investigation. It still mysterious to me why it doesn't create warning, even if I have added `return 11` at the end of this function.

Reply:
Please just create an issue. I don't think this is a good way to set a reminder :)

Reply:
:+1: